generators/app/templates/src/app/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+interface Props {
+  buttonClassName?: string;
+  className?: string;
+  formClassName?: string;
+  submitButtonText: string;
+  children: React.ReactNode;
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+}
+
+function SearchBar({
+  submitButtonText,
+  className = '',
+  formClassName,
+  buttonClassName = '',
+  onSubmit,
+  children
+}: Props) {
+  return (
+    <div className={className}>
+      <form onSubmit={onSubmit} className={formClassName}>
```

Comment:
Are we keeping the outer `div` just to apply some styles? Can't we join the `div` and `form` styles into one `className` and remove the outer `div`?